app/browser/api/ledger.js

```diff
@@ -2254,6 +2254,13 @@ const deleteSynopsis = () => {
   synopsis.publishers = {}
 }
 
+const yoDawg = (stateState) => {
```

Comment:
💯

Reply:
Maybe we can put a comment above this just describing why this is needed- for example:
```
// fix for incorrectly persisted state (see https://github.com/brave/browser-laptop/issues/11585)
```

Reply:
good call :+1: